packages/create-skill-app/index.ts

```diff
@@ -47,6 +48,75 @@ async function run(): Promise<void> {
     }
   }
 
+  const questions: PromptObject[] = [
+    {
+      type: 'text',
+      name: 'projectTitle',
+      message: `What is the product's title?`,
+      initial: 'Skill Product Title',
+    },
+    {
+      type: 'number',
+      name: 'devPort',
+      message: `What http port will this run on localhost?`,
+      min: 3000,
+      max: 3999,
+      initial: 3000,
+    },
+    {
+      type: 'number',
+      name: 'databasePort',
+      message: `What port will the MySQL database run on in dev?`,
+      min: 3306,
+      max: 3999,
+      initial: 3306,
+    },
+    {
+      type: 'text',
+      name: 'firstName',
+      message: `What is our partner's first name?`,
+      initial: 'First',
+    },
+    {
+      type: 'text',
+      name: 'lastName',
+      message: `What is our partner's first name?`,
+      initial: 'Last',
+    },
+    {
+      type: 'text',
+      name: 'twitter',
+      message: `What is our partner's twitter? (no @)`,
+      initial: 'twitterHandle',
+    },
+    {
+      type: 'text',
+      name: 'domainHost',
+      message: `What is the host name this will be deployed to? (no https:// or www)`,
+      initial: 'example.com',
+    },
+    {
+      type: 'text',
+      name: 'description',
+      message: `How would you describe this product to a colleague?`,
+      initial: 'Deep dive workshop to learn about interesting things.',
+    },
+    {
+      type: 'text',
+      name: 'keywords',
+      message: `What keywords describe this product? (comma seperated)`,
+      initial: 'web development, programming, javascript',
+    },
+    {
+      type: 'text',
+      name: 'nextAuthDevSecret',
+      message: `next-auth secret (only for dev)`,
+      initial: crypto.randomBytes(32).toString('hex'),
+    },
+  ]
+
+  const projectData = await prompts(questions)
```

Comment:
makes it ez to add additional questions and the `prompts` lib has a lot of flexible options. If you just mash enter all the way through everything works fine.